napari/util/icons.py

```diff
@@ -0,0 +1,66 @@
+"""Utility script to generate copies of icons with colors based
```

Comment:
would prefer for this to be under `resources`

Reply:
great call, moved, and also renamed to `build_icons`